pkg/tests/lib/test-admin.js

```diff
@@ -20,6 +20,10 @@ class TestAdmin extends Xen.Async {
     };
   }
   async onTestOptionClick(e) {
+    // Reset result.
+    this.state.result = undefined;
```

Comment:
The original intent was changing state will cause invalidation, so the typical pattern (for XenAsync) here is:
`this.mergeState({result: null});`

Also, I never assign 'undefined', only 'null'.

Reply:
Fwiw, I have a workstream to make XenPlus or something where the lifecycle is more like Particles.

Reply:
Ah thank you! Nice reminder on `mergeState`. Updated.